src/main/java/nextstep/courses/domain/Image.java

```diff
@@ -0,0 +1,32 @@
+package nextstep.courses.domain;
+
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import javax.imageio.ImageIO;
+import org.springframework.core.io.ClassPathResource;
+
+public class Image {
+
+    private final String imageUrl;
+    private final ImageMeta imageMeta;
+
+    public Image(String imageUrl, int width, int height, int fileSize, String ext) {
+        this.imageUrl = imageUrl;
+        this.imageMeta = new ImageMeta(width, height, fileSize, ext);
```

Comment:
```suggestion
       this(imageUrl, new ImageMeta(width, height, fileSize, ext);
```

인스턴스 변수와 같은 타입을 가지는 주 생성자를 추가하고 위와 같이 구현해 보는 것은 어떨까?

Reply:
[d563e3d](https://github.com/next-step/java-lms/pull/503/commits/d563e3d8b9a87a8b94b5488b5d599473c83bd1f6) 이런게 반영했습니다!